common/autoware_auto_perception_rviz_plugin/src/object_detection/detected_objects_display.cpp

```diff
@@ -53,6 +53,28 @@ void DetectedObjectsDisplay::processMessage(DetectedObjects::ConstSharedPtr msg)
       label_marker_ptr->id = id++;
       add_marker(label_marker_ptr);
     }
+
+    // Get marker for velocity text
+    geometry_msgs::msg::Point vel_vis_position;
+    vel_vis_position.x = object.kinematics.pose_with_covariance.pose.position.x - 0.5;
```

Comment:
why x is substracted? :pray:

Reply:
I think that's right.
The main reason why it is substracted is to alien visualization of TrackedObjects.
<https://github.com/autowarefoundation/autoware.universe/blob/main/common/autoware_auto_perception_rviz_plugin/src/object_detection/tracked_objects_display.cpp#L60>

Reply:
I understand that it is a separate PR. :+1: 
can you add todo in source code?
@miursh